docs/sdk/cosmos-sdk-cli.md

```diff
@@ -0,0 +1,34 @@
+# cosmos-sdk-cli 
+Create a new blockchain project based on cosmos-sdk with a single command.
+
+---
+
+# Installation
+
+```shell
+go get github.com/cosmos/cosmos-sdk
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+make all
+```
+
+This will install a binary cosmos-sdk-cli
+
+# Creating a new project
+
+**cosmos-sdk-cli init** _Your-Project-Name_
+
+This will initialize a project, the dependencies, directory structures with the specified project name.
+
+### Example:
+```shell
+cosmos-sdk-cli init testzone -p github.com/your_user_name/testzone
```

Comment:
Can you prefix with a `$ `, just to indicate this command is being ran in a shell. (Not a huge deal, I think its just the standard we've adopted. I could be wrong on this point)

Reply:
Will do